<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Desde</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dBListMin"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Hasta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dBListMax"/>
     </item>
     <item>
      <widget class="QPushButton" name="bBListShow">
       <property name="text">
        <string>Mostrar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bExportDraft">
       <property name="text">
        <string>Exportar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tBList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount">
      <number>10</number>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NÃºmero</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cliente</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IVA</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Items</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Usuario</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
